google-cloud-examples/src/main/java/com/google/cloud/examples/compute/snippets/CreateAddressDiskAndInstance.java

```diff
@@ -43,14 +41,12 @@
   private static final String ADDRESS_NAME = "test-address";
   private static final String DEFAULT_IMAGE =
       "https://www.googleapis.com/compute/v1/projects/debian-cloud/global/images/debian-7-wheezy-v20150710";
```

Comment:
Since this is a sample, opposed to a Test, I wonder if you might be better off using String.format and fill in the blanks, so users can update to their choice easily. (also since this doesn't get updated by RenovateBot).   FYI Debian is currently at 10, so please update